docs/advanced/integration_tests.md

```diff
@@ -0,0 +1,130 @@
+# Integration tests (advanced)
```

Comment:
I wouldn't call them advanced - they are basically simplified for different use cases, so they are actually easier to use, right?

Reply:
So instead of "advanced" we call them "advanced topics"... Which is basically the same? I still don't get what makes these tests so advanced. It's not like they are in some way complex or anything...